README.md

```diff
@@ -460,6 +494,27 @@ cygnusagent.sinks.mysql-sink.mysql_password = xxxxxxxxxxxx
 # how the attributes are stored, either per row either per column (row, column)
 cygnusagent.sinks.mysql-sink.attr_persistence = column
 
+# ============================================
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/6133c3124462f6bac841b3fc6e4fc099baab4cf6